models/tokens/ethereum/tokens_ethereum_transfers_base.sql

```diff
@@ -0,0 +1,22 @@
+{{ config(
+        alias = 'transfers_base',
+        tags=['dunesql'],
+        partition_by = ['token_standard', 'block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['block_number', 'tx_index', 'evt_index', 'trace_address'],
```

Comment:
are we missing `tx_hash` as a unique key? or does introducing `tx_index` make it unique without the hash?

Reply:
`tx_index` is equivalent to `tx_hash` here (combined with `block_number`) so it's not needed, this should be more efficient.

Reply:
looks like maybe we need an additional column in unique keys, based on test failure?

Reply:
Yeah I am adding it at the moment. I think the nulls are causing issues with the merge into.

Reply:
No duplicates now. I'm not sure native transfers are 100% correct though (aside from gas) so will need to test this further later.